CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 1.1.0 (2021-11-11)
+
+
+### Features
+
+* context destroy --force flag ([#118](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/issues/118)) ([4282093](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/commit/428209311aa247c999816348a972737739b1189f))
+* Initial infrastructure for MiniWdl support ([#125](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/issues/125)) ([387393a](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/commit/387393a64593d08ec2016a69382f7d64de37914e))
+* Introducing AWS Lambda based WES Adapter for running the workflows ([#155](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/issues/155)) ([bd90f47](https://codestar-connections.us-west-2.amazonaws.com/git-http/680431765560/us-west-2/d075f301-104b-41c2-9281-8705914f195b/aws/amazon-genomics-cli/commit/bd90f47b94ede34c31ea109221225ff3cd65d200))
```

Comment:
Are these links correct? I'd think we'd just want the github URLs, not the CodeStar connections.

Reply:
Good catch! They are not correct... I'm not sure why it's linking the codestar connection rather than github directly. It may be due in part to the cloning